### DISCUSSION TITLE
Is there a way to update validation rules?

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#320] [Questions] [answered]
I have a custom phone input component and I need to update the min rule to match the length count of a mask I retrieve when switching countries. Is there a way to change the value of the min rule from the component?

**Comment by adamberecz** [comment, marked as answer]
I'm not sure if this will any side effects but this is how you could do it:
```js
const form = ref({
  schema: {
    phone: {
      type: 'phone',
      rules: 'min:10',
      onMounted(el$) {
        el$.options$.on('select', () => {
          el$.Validators[0].attributes[0] = 12
        })
      }
    },
  },
})
```
